src/platform_impl/linux/x11/window.rs

```diff
@@ -1134,13 +1147,13 @@ impl UnownedWindow {
         let cursor = unsafe {
             // We don't care about this color, since it only fills bytes
             // in the pixmap which are not 0 in the mask.
-            let dummy_color: ffi::XColor = mem::uninitialized();
```

Comment:
Why not `mem::zeroed()` and be done with it?

Reply:
Same as above, the value is not used.